packages/falso/src/lib/credit-card-number.ts

```diff
@@ -12,19 +27,38 @@ import { randNumber } from './number';
  *
  * @example
  *
+ * randCreditCardNumber({ issuer: 'Visa'})
+ *
+ * @example
+ *
  * randCreditCardNumber({ length: 10 })
  *
  */
-export function randCreditCardNumber<Options extends FakeOptions>(
-  options?: Options
-) {
+
+export function randCreditCardNumber<
+  Options extends FakeOptions & {
+    issuer?: Issuer;
+  }
+>(options?: Options) {
   return fake(() => {
-    let cardNumber = '';
+    let card = rand(data);
```

Comment:
We can skip this function when we have issuer & issuerCard

Reply:
Usage of fake function is removed @NetanelBasal